### DISCUSSION TITLE
how to start `grep_string` without scoping on current word

### DISCUSSION THREAD

**Discussion by vitaly** [nvim-telescope/telescope.nvim#1556] [Q&A] [answered]
When I do `:Telescope grep_string` it starts already scoped with a search of the current word: `Find Word (foobar)`.
Is there a way to prevent this?

**Comment by Conni2461** [comment, marked as answer]
You can just pass your own search query. `:Telescope grep_string search=xyz`. `:help builtin.grep_string` for more information

**Reply by vitaly** [reply]
I don't want to pass query, I want to type my query in the prompt. but passing en empty one worked! Thanks!

**Reply by Conni2461** [reply]
`live_grep` also exists. It sends the prompt after every prompt change to ripgrep

**Reply by vitaly** [reply]
`live_grep` is not fuzzy ;)
